### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by husseinTalal2**
the footer is finished with the tests, but i didn't added the paths of the links because we haven't created the pages yet

### CODE REVIEW

File: src/containers/Footer/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@import '../../styles/variables';
+
+footer {
+  background-color: $darkerBlue !important;
+  min-height: 250px;
+  .container {
```
Comment (MustafaNawzat) [inline src/containers/Footer/Footer.scss:6]: try not to overwrite bootstrap classes, instead make your own class and play with it.

File: src/containers/Footer/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    min-height: 250px;
+    .row {
+      min-height: 250px;
+    }
+  }
```
Comment (MustafaNawzat) [inline src/containers/Footer/Footer.scss:11]: leave an empty line

File: src/containers/Footer/Footer.scss (hunk 1 above)
Comment (husseinTalal2) [reply src/containers/Footer/Footer.scss:6]: do you mean not to use `!important ` to overwrite bootstrap classes ?

Comment (MustafaNawzat) [reply src/containers/Footer/Footer.scss:6]: No, I mean like here you changed the `min-height` by targeting the bootstrap class, I would recommend if you modify a new class for that. for ex:

```
.footerContainer{
    min-height: 250px;
}
```

**Pull request merged**